cpp/oneapi/dal/algo/pca/test/batch.cpp

```diff
@@ -27,6 +27,9 @@ namespace oneapi::dal::pca::test {
 namespace te = dal::test::engine;
 namespace la = te::linalg;
 
+const std::string datasets = std::getenv("DATASETS");
```

Comment:
```suggestion
const std::string datasets = std::getenv("DATASETSROOT");
```

Reply:
I suggest to check if variable exists. It it's not, `getenv` return `nullptr`. I'd report this to user via exception and require to set this var.